linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java

```diff
@@ -46,7 +46,9 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
 			Arguments.of("amazonlinux", "2"),
 			Arguments.of("centos", "7"),
 			Arguments.of("registry.access.redhat.com/ubi7/ubi", "latest"),
-			Arguments.of("oraclelinux", "7")
+			Arguments.of("redhat/ubi8", "latest"),
+			Arguments.of("oraclelinux", "7"),
+			Arguments.of("oraclelinux", "8")
```

Comment:
Can also add 'redhat/ubi9' for  'rpm/rhel/9'?

Reply:
in theory, we do not need to test the same OS for both yum and dnf. some old ones, e.g ubi7 does not support dnf so it has to be in  yum test

I intended to remove oraclelinux 8 and ubi8 from Yum as well (already covered in dnf)